app/controllers/users_controller.rb

```diff
@@ -53,6 +53,20 @@ def update
     render :show, status: :internal_server_error
   end
 
+  def show_remove_user
+    @user = as_team_member(cognito_user: get_user(user_id: params[:user_id]))
+  end
+
+  def remove_user
+    @user = as_team_member(cognito_user: get_user(user_id: params[:user_id]))
+    delete_user(username: @user.email)
+    UserDeletedEvent.create(data: { username: @user.email, user_id: params[:user_id], name: @user.given_name })
+    redirect_to users_path
+  rescue AuthenticationBackend::AuthenticationBackendException
+    flash.now[:errors] = t('users.remove_user.errors.generic_error')
+    render :show, status: :bad_request
```

Comment:
i don't think you can just render `show` it needs other instance variables, either drop `now` from the flash and do redirect to users_path, or maybe render the show_remove_user?